docs/code-of-conduct.rst

```diff
@@ -0,0 +1,33 @@
+Code of Conduct
+===============
+
+Like the technical community as a whole, the Write the Docs team and community is made up of a mixture of professionals and volunteers from all over the world, working on every aspect of the mission - including mentorship, teaching and connecting people.
```

Comment:
I agree on the question about the inclusion of "mission" in regards to community. Maybe we can strike out the rest of the sentence after "world".

On the "professionals and volunteers" phrase, you could count user groups of proprietary software as volunteer efforts. I'm more inclined to let that phrase stay.